app/models/surgeon.rb

```diff
@@ -13,6 +15,14 @@ class Surgeon < ActiveRecord::Base
   validates :last_name, uniqueness: { scope: :first_name }
   validates :last_name, presence: true
 
+  def full_name
+    "#{last_name}-#{first_name}"
+  end
+
+  def satisfaction_by_procedure
```

Comment:
could also `.group([:procedure])` to create hash of actual procedure object => sat score, but grabbing those procedures down the line seemed ok